packages/webapp/src/containers/Finances/ActualRevenue/index.jsx

```diff
@@ -49,9 +51,9 @@ export default function ActualRevenue({ history, match }) {
   );
 
   useEffect(() => {
-    if (!allRevenueTypes?.length) {
-      dispatch(getRevenueTypes());
-    }
+    dispatch(getRevenueTypes());
```

Comment:
An inelegant but working solution seems to be just calling `fetchAllData()` here instead 😬

Reply:
So I have no idea why, but the getSales action wasn't actually working -- setting it up with `createAction` as some of the others seems to have done the trick